integreat_cms/static/src/js/forms/update-permalink.ts

```diff
@@ -40,21 +40,25 @@ window.addEventListener("load", () => {
             .setAttribute("data-copy-to-clipboard", encodeURI(updatedLink.concat(currentSlug)));
     };
 
-    if (
-        document.getElementById("id_title") &&
-        (document.querySelector('[for="id_title"]') as HTMLElement)?.dataset?.slugifyUrl
-    ) {
-        document.getElementById("id_title").addEventListener("focusout", ({ target }) => {
-            const submissionLock = new SubmissionPrevention(".no-premature-submission");
-            const currentTitle = (target as HTMLInputElement).value;
-            const dataset = (document.querySelector('[for="id_title"]') as HTMLElement).dataset;
-            slugify(dataset.slugifyUrl, { title: currentTitle, model_id: dataset.modelId })
-                .then((response) => {
-                    /* on success write response to both slug field and permalink */
-                    slugField.value = response.unique_slug;
-                    updatePermalink(response.unique_slug);
-                })
-                .finally(() => submissionLock.release());
+    if (document.querySelectorAll("#id_title")) {
```

Comment:
I think you can remove this line (and the cloding bracket, of course) - if there is no `#id_title`, then `document.querySelectorAll("#id_title").forEach((item) => { ... }` will simply run 0 times.

Reply:
Thank you a lot for your feedback! This is should be resolved now! :blush: